mapping.json

```diff
@@ -0,0 +1,111 @@
+{
+    "schema_id": "cof_r2_assessment",
+    "unscored_assessment_sections": [{
+        "id": "org_info",
+        "name": "Organisation information",
+        "sub_criteria": [{
+            "id": "org_info",
+            "name": "Organisation information",
+            "themes": [{
+                "id": "general_info",
+                "name": "General information",
+                "answers": [{
+                    "field_id": "pipopl",
+                    "field_type": "textarea"
```

Comment:
Could this be derived from the `type` field in the submitted JSON? So maybe we don't need it here as well?

Reply:
I had this discussion with @stevewalton28 and @harryyo yesterday. 
The reason we kept it in is to have a simple toggle when deciding how to render the field, the runner might say it's an XX type, but we may want to display it in a text field template etc. If I have understood correctly, It gives more responsibility to the assessment side in presentation

Reply:
OK makes sense.